ueransim_lb/ueransim-gnb_init.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# UERANSIM gNB initialization script for NGAP load balancing
+# Connects to loxilb VIP instead of direct AMF
+
+export IP_ADDR=$(awk 'END{print $1}' /etc/hosts)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)